content/installation/python/PythonConfig.md

```diff
@@ -5,6 +5,7 @@ tags: "installation python django flask pyramid agent service configuration"
 -->
 
 The Python agent and Contrast Service use a YAML file to alter the agent behavior. Go to the Python [YAML Template](installation-pythonconfig.html#python-template) for fully formatted properties that you can copy and use in your own configuration files.
+Environment variables may also be used to override any configuration in the YAML file.
```

Comment:
Please write in active voice:

You can also use environment variables to override any configuration...